espnet/utils/fill_missing_args.py

```diff
@@ -7,7 +7,7 @@
 import logging
 
 
-def fill_missing_args(args, add_arguments):
+def fill_missing_args(args, add_arguments, dummy_input=None):
```

Comment:
dummy_input does not make sense.
cmd_line_args? not so good :(
Or if you use this method for only build method, revert this change and then add some operations in build method.

Reply:
pre_parse_str ?

Reply:
Now I understand why this option is needed.
Why don’t you remove required=True instead pf this option?